poll/public/handlebars/poll_results.handlebars

```diff
@@ -1,13 +1,13 @@
 <script id="poll-results-template" type="text/html">
     <h3 class="poll-header">{{display_name}}</h3>
     <div class="poll-question-container">{{{question}}}</div>
-    <ul class="poll-answers-results poll-results">
+    <ul class="poll-answers-results poll-results {{~#if any_img}} has-images{{/if}}">
```

Comment:
@Kelketek I only learned this when doing this PR, but apparently the tilde removes the whitespace before the template tag, when underscore renders the template. You can put a tilde on the left `{{~#if blah}}` and/or right `{{#if blah~}} ` of these handlebars tags, to remove the whitespace on either side or both.